files/default/neutron-ha-tool.py

```diff
@@ -248,6 +254,11 @@ def run(args):
     else:
         raise ValueError('Invalid agent_selection_mode')
 
+    if args.target_agent_id:
+        agent_picker = AgentIdBasedAgentPicker(args.target_agent_id)
+    if args.target_host:
```

Comment:
If both are specified, `--target-agent-id` will be ignored, which is unintuitive.  If you `add_mutually_exclusive_group` above then that defends against this, but it would still be better to change this to `elif`.